src/core/test/mock/pile.sol

```diff
@@ -18,6 +18,7 @@ contract PileMock {
 
     // returns
     uint public debtReturn; function setLoanDebtReturn(uint debtReturn_) public {debtReturn=debtReturn_;}
+    uint public debtOfReturn; function setDebtOfReturn(uint debtOfReturn_) public {debtOfReturn=debtOfReturn_;}
```

Comment:
why not use setLoanReturn? or if we want to be more modular, completely break up setLoanReturn into the three individual params on the loan.

Reply:
because we currently using the following pattern:
the original contract as a method like
`function X(...) returns(uint)`

the mock contract has a method called 
`function setXReturn(uint)`